### PR TITLE
Launcher args handling

### LINKED ISSUE #81: Invoking LayrryLauncher with no args should print usage and quit
Currently it prints an exception and quits

```sh
Exception in thread "main" com.beust.jcommander.ParameterException: The following option is required: [--layers-config]
	at com.beust.jcommander.JCommander.validateOptions(JCommander.java:388)
	at com.beust.jcommander.JCommander.parse(JCommander.java:357)
	at com.beust.jcommander.JCommander.parse(JCommander.java:335)
	at org.moditect.layrry.launcher.LayrryLauncher.launch(LayrryLauncher.java:50)
	at org.moditect.layrry.launcher.LayrryLauncher.main(LayrryLauncher.java:41)
```

### LINKED ISSUE #83: LayrryLauncher shoud print out supported config formats
As part of the help usage.

### PR DESCRIPTION
**Pull request by aalmiray**
Fixes #81
Fixes #83

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Could you prefix the commit messages with the issue keys ("#81 Fixing...")?

File: layrry-launcher/src/main/java/org/moditect/layrry/launcher/internal/Args.java [hunk 1]
```diff
@@ -26,7 +26,7 @@
     @Parameter(description = "[arguments]")
     private List<String> mainArgs = new ArrayList<>();
 
-    @Parameter(names = "--layers-config", required = true, description = "Layers configuration file. May be a local file or remote URL.")
+    @Parameter(names = "--layers-config", description = "Layers configuration file. May be a local file or remote URL.")
```
Comment (gunnarmorling) [inline layrry-launcher/src/main/java/org/moditect/layrry/launcher/internal/Args.java:29]: Can we add some examples perhaps? Not sure what's the best place, the option description itself, or maybe the overall tool description (assuming there is such thing)?

Comment (aalmiray) [reply layrry-launcher/src/main/java/org/moditect/layrry/launcher/internal/Args.java]: Done. Usage now looks like

```sh
Missing --layers-config parameter or local file named layers[.yml|.toml]

Usage: org.moditect.layrry.launcher.LayrryLauncher [options] [arguments]
  Options:
    --basedir
      Base directory for resolving relative paths
    --help
      Show usage and quit
    --layers-config
      Layers configuration file. May be a local file or remote URL. Examples: 
      path/to/layers.yml, https://server/path/to/layers.toml
    --properties
      Additional configuration properties. May be a local file or remote URL. 
      Must use the Java properties format. Examples: 
      path/to/versions.properties, https://server/path/to/versions.properties
    --version
      Show version and quit

Supported config formats are [.yml, .toml]
```

Comment (gunnarmorling) [reply layrry-launcher/src/main/java/org/moditect/layrry/launcher/internal/Args.java]: A, that's perfect!

Comment (gunnarmorling) [reply layrry-launcher/src/main/java/org/moditect/layrry/launcher/internal/Args.java]: The one thing we might want to revise later on is the FQN right at the top, as that'd be confusing when e.g. used via sdkman or jbang. But we can leave that for another time.

**Comment by gunnarmorling** [comment]
Applied; thx!

File: layrry-launcher/src/main/java/org/moditect/layrry/launcher/internal/Args.java (hunk 1 above)
Comment (aalmiray) [reply layrry-launcher/src/main/java/org/moditect/layrry/launcher/internal/Args.java]: The FQN was added recently with https://github.com/moditect/layrry/commit/47a4d851dd5cbd0fb0b9117a51002bdeca4034fd as it displayed `<main-class>` before.

**Pull request merged**